addons/mail/models/mail_mail.py

```diff
@@ -292,12 +292,14 @@ def _send_prepare_values(self, partner=None):
         if partner:
             emails_normalized = tools.email_normalize_all(partner.email)
             if emails_normalized:
+                name = partner.name.replace('@', '') or "False"
                 email_to = [
-                    tools.formataddr((partner.name or "False", email or "False"))
+                    tools.formataddr((name, email or "False"))
                     for email in emails_normalized
                 ]
             else:
-                email_to = [tools.formataddr((partner.name or "False", partner.email or "False"))]
+                name = partner.name.replace('@', '') or "False"
```

Comment:
this line is redundant, we can create a variable just bellow `emails_normalized = `